sonic_data_client/db_client.go

```diff
@@ -80,6 +80,23 @@ var IntervalTicker = func(interval time.Duration) <-chan time.Time {
 	return time.After(interval)
 }
 
+// Define a mutex to synchronize access to the IntervalTicker variable
+var intervalTickerMutex sync.Mutex
+
+// Define a new function to set the IntervalTicker variable
+func SetIntervalTicker(f func(interval time.Duration) <-chan time.Time) {
```

Comment:
>SetIntervalTicker [](http://example.com/codeflow?start=5&length=17)

Production code does not call `SetIntervalTicker` ?

Reply:
yes, original product codes use the default global function, just server_test code reset the function and will use SetIntervalTicker